pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 100
```

Comment:
In addition to this setting I use (as we are on Python 3.6+):

```
[tool.black]
line-length = 100
target-version = ["py36", "py37"]
skip-numeric-underscore-normalization = true
```

`skip-numeric-underscore-normalization` is up for debate of course, but I don't like underscores in numbers at all..

Reply:
Just noticed that underscore normalization has been removed from black altogether. There is no need to include the flag anymore.